AbilityChanger/Abilities/Scream.cs

```diff
@@ -53,12 +125,24 @@ public override void OnFsmEnable(On.PlayMakerFSM.orig_OnEnable orig, PlayMakerFS
             }
         }
 
+        public void HandleStart()
+        {
+            currentAbility.Start();
+        }
 
+        public void HandleTrigger()
+        {
+            currentAbility.Trigger("BUTTON DOWN");
+        }
 
+        public void HandleComplete()
+        {
+            currentAbility.Complete(false);
+        }
 
-
-
-
-
+        public void HandleSpawn()
```

Comment:
what is the difference between Trigger and Spawn for us? is there something that triggers first and spawns later in vanilla ?